src/api/app/controllers/webui/package_controller.rb

```diff
@@ -157,6 +157,7 @@ def binary
   def binaries
     @repository = params[:repository]
     @package_name = params[:package]
```

Comment:
view or helper call a presenter. Not the controller. https://www.rubyguides.com/2019/09/rails-patterns-presenter-service/

Reply:
Fixed